modules/ROOT/pages/amazon/amazon-sqs-connector-design-center.adoc

```diff
@@ -0,0 +1,68 @@
+= Amazon SQS Design Center Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+The Flow Designer feature of Design Center enables you to create apps visually. To use Flow Designer, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the
+xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Flow Designer:
+
+* Configure the input source (trigger) for your app.
+* Configure the target component for your app.
+
+For information on Flow Designer field values, see
+the xref:amazon/amazon-sqs-connector-reference.adoc[Amazon SQS Connector Reference].
+
+== Configure the Input Source Trigger
+
+. In Design Center, click *Create*.
+. Click *Create new application* to open Flow Designer.
+. Specify a value for *Project name*, and click *Create*.
+. Click *Go straight to canvas* to exit from *Let's get started*.
+. Click on the trigger card.
+. Search for the connector.
+. Select the connector name.
+. Select the *Recieve messages* operation to use it as a source. (You can also use the HTTP Connector or Scheduler component as a source.)
+. Click *Click here to set it up*.
+. Enter values for these fields:
++
+[%header,cols="30s,70a",width=80%]
+|===
+|Parameter |Description
+|*Configuration Name* |Name used to reference the configuration
+|*Connection* | Connection type (`Basic` or `Role`)
+|*Access Key* |Alphanumeric text string that uniquely identifies the user who owns the account
+|*Secret Key* |Key that plays the role of a password
+|===
++
+. Click *Test* to confirm that the parameters of your global configuration are accurate, and that Mule is able to successfully connect to your instance of Amazon SQS.
+. Click off the trigger card.
++
+Flow Designer automatically saves all changes.
+
+== Configure the Target Component
+
+. Click *+* next to the trigger card.
+. In *Select a component*, search for the connector name.
+. Select the connector name.
+. Select an operation for the connector, such as *Get Queue Attributes*.
+. If needed, specify values that are required in the *General* tab. For the *Get Queue Attributes* operation, enter the Amazon SQS queue URL in the *Queue URL* field.
+. Select *Edit* to set up other required fields.
++
+For the *Get Queue Attributes* operation, the required fields are as follows. Depending on the trigger you used, these fields might already have values.
```

Comment:
Depending on the trigger you used, the following required fields for the Get Queue Attributes operation might already have values.